public/pages/CreateTrigger/containers/CreateTrigger/CreateTrigger.js

```diff
@@ -74,14 +75,12 @@ export default class CreateTrigger extends Component {
       ...uiMetadata,
       triggers: { ...uiMetadata.triggers, ...triggerMetadata },
     };
-    updateMonitor({ triggers: updatedTriggers, ui_metadata: updatedUiMetadata })
+    const actionKeywords = ['create', 'trigger'];
+    updateMonitor({ triggers: updatedTriggers, ui_metadata: updatedUiMetadata }, actionKeywords)
       .then((res) => {
         setSubmitting(false);
         if (res.ok) {
           onCloseTrigger();
-        } else {
-          console.log('Failed to create the trigger:', res);
-          this.backendErrorHandler('create', res);
```

Comment:
Why is this being removed?

Reply:
I added the code of showing notifications in another function:
https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/blob/a8350e0f6842fcea59a41754c10a6648df392286/public/pages/MonitorDetails/containers/MonitorDetails.js#L140,
which is the function being called eventually, and the only function being called for the "Disable/Enable" button in "monitor details" page, so I remove the part of code here to avoid duplication.